src/components/Routes.js

```diff
@@ -3,11 +3,15 @@ import { Route, Switch } from 'react-router-dom';
 import NotFound from './notfound/notFound';
 import Welcome from './welcome/welcome';
 import Privacy from './privacy/privacy';
+import Signup from './signup';
+import EmailVerification from './emailVerification/emailVerification';
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={Welcome} />
     <Route exact path="/privacy" component={Privacy} />
+    <Route exact path="/signup" component={Signup} />
+    <Route path="/api/v1/users/verify" component={EmailVerification} />
```

Comment:
This path is so long. It would be good if you make it short like ```/verifyEmail``` instead of using the path of the api